etc/kubernetes/operator-debezium/lgarciaac/app-config/application.properties

```diff
@@ -9,4 +9,4 @@ quarkus.log.category."io.fabric8.kubernetes.client.internal.VersionUsageUtils".l
 # operator
 #
 cos.connectors.namespace = ${kubernetes.namespace}
-cos.operators.namespace   = ${kubernetes.namespace}
\ No newline at end of file
+cos.operators.namespace   = ${kubernetes.namespace}
```

Comment:
I am wondering about this change. What exactly changed here?
Can you maybe add a commented line with the new config property and a placeholder value?